packages/core/src/languages/solidity/predeploys.ts

```diff
@@ -44,6 +48,48 @@ export const ensureSphinxAndGnosisSafeDeployed = async (
   await deploySphinxSystem(provider, wallet, relayers, executionMode, logger)
 }
 
+export const cancelPreviousDripVersions = async (
```

Comment:
Handles archiving any previously created Drippie drips before the new version is created.